website/docs/docs/build/packages.md

```diff
@@ -157,9 +157,60 @@ packages:
 
 Where `name: 'dbt_utils'` specifies the subfolder of `dbt_packages` that's created for the package source code to be installed within.
 
-### Private packages
+## Private packages
 
-#### SSH Key Method (Command Line only)
+### Native private packages <Lifecycle status='beta'/> 
+
+dbt Cloud supports private packages from [supported](#prerequisites) Git repos leveraging an exisiting [configuration](/docs/cloud/git/git-configuration-in-dbt-cloud) in your environment. Previously, you had to configure a [token](#git-token-method) to retrieve packages from your private repos.  
+
+#### Prerequisites
+
+Native support for private packages is supported on the following services:
```

Comment:
```suggestion
Native support for private packages is supported for the following providers:
```

Reply:
The providers don't support it in this scenario though. I'll change it to 
```suggestion
dbt Cloud natively supports private packages for the following services:
```